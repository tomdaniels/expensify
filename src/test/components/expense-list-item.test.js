import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/expense-list-item';
import expenses from '../fixtures/expenses';

test('should render expenses data', () => {
    const wrapper = shallow(
        <ExpenseListItem
            id={expenses[0].id}
            description={expenses[0].description}
            amount={expenses[0].amount}
            createdAt={expenses[0].createdAt}
        />);
    expect(wrapper).toMatchSnapshot();
});