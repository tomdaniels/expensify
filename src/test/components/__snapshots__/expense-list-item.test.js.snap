// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expenses data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    gum
  </Link>
  <p>
    $1.95
    -
    1st January, 1970
  </p>
</div>
`;
